src/ast/mod.rs

```diff
@@ -2713,6 +2752,25 @@ impl fmt::Display for Statement {
                 if let Some(order_by) = order_by {
                     write!(f, " ORDER BY ({})", display_comma_separated(order_by))?;
                 }
+                if let Some(bigquery_config) = big_query_config {
```

Comment:
The names `bigquery_config` and `big_query_config` are different, is that intentional?

Reply:
Ah right that wasn't intentional, updated!